test/support/application_fixture_helper.rb

```diff
@@ -0,0 +1,75 @@
+# typed: ignore
+# frozen_string_literal: true
+
+module ApplicationFixtureHelper
+  TEMP_FIXTURE_DIR = ROOT.join("tmp", "fixtures").to_s
+  DEFAULT_TEMPLATE = :minimal
+
+  def setup_application_fixture
+    @old_working_dir = Dir.pwd
+  end
+
+  def teardown_application_fixture
+    Dir.chdir(@old_working_dir)
+    FileUtils.remove_entry(@app_dir, true) if defined? @app_dir
+  end
+
+  def copy_template(template)
+    copy_dir("test/fixtures/#{template}")
+    Dir.chdir(app_dir)
+  end
+
+  def app_dir
+    copy_template(DEFAULT_TEMPLATE) unless defined? @app_dir
+    @app_dir
+  end
+
+  def config
+    @config ||= Packwerk::Configuration.from_path(app_dir)
+  end
+
+  def path_to(relative_path)
```

Comment:
I don't see this method being used other than in this module. I think it would make sense to make it private.

Reply:
Just saw that it is used in the next commit.